samples/drivers/display/sample.yaml

```diff
@@ -1,6 +1,10 @@
 sample:
   description: Sample application for displays
   name: display_sample
+common:
+  harness: console
+  harness_config:
+    fixture: fixture_display
```

Comment:
Does this play nicely with the native_posix target, if not we will lose coverage in CI?

Reply:
Just double checked this change prevents execution of the test case on the native_posix board.